test/backend/routes.spec.js

```diff
@@ -262,6 +315,9 @@ describe('Routes - ', () => {
         .catch(done);
     });
 
+    // This tests sets the cookies required for user authentication, so
+    // all the tests requiring user authentication should be put placed
+    // after this one.
```

Comment:
🐮  comment style with `/*`

Reply:
could we move this into a `beforeEach`? otherwise, it'll be impossible to test an individual test